File: Java+/Spring+/spring-java13-example/src/main/java/app/mvc/HelloWorldController.java
package app.mvc;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.ResponseBody;

@Controller
public class HelloWorldController {
    static final String ENDPOINT = "/hello";
    static final String CONTENT = "Hello, World!";

    @ResponseBody
    @GetMapping(ENDPOINT)
    public String hello() {
        return CONTENT;
    }
}

File: Kotlin+/spring-kotlin-java12-maven-example/src/main/kotlin/app/mvc/HelloWorldController.kt
package app.mvc

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseBody


@Controller
class HelloWorldController {
    companion object {
        const val ENDPOINT = "/hello"
        const val CONTENT = "Hello, World!"
    }

    @ResponseBody
    @GetMapping(path = [ENDPOINT])
    fun greeting(): String {
        return CONTENT
    }
}